build-system/common/check-package-manager.js

```diff
@@ -63,34 +63,36 @@ function yellow(text) {
 }
 
 /**
- * @fileoverview Perform checks on the AMP toolchain.
- */
-
-// If npm is being run, print a message and cause 'npm install' to fail.
-function ensureYarn() {
-  if (process.env.npm_execpath.indexOf('yarn') === -1) {
+ * If yarn is being run, print a message and cause 'yarn install' to fail.
+ **/
+function ensureNpm() {
+  if (process.env.npm_execpath.indexOf('npm') === -1) {
     console.log(
-      red('*** The AMP project uses yarn for package management ***'),
+      red('*** The AMP project now uses npm for package management ***')
```

Comment:
Since this is one long constant string, I'd like to suggest making it a module-level constant:

```javascript
const NPM_INFO_MSG = `${red('*** The AMP project now uses npm for package management ***')}
For more info, see ${
  cyan('https://github.com/ampproject/amphtml/issues/30518')
}.

${yellow('To install all packages:')}
${cyan('$')} npm install

${yellow('To install a new (runtime) package to "dependencies":')}
${cyan('$')} npm install [package_name@version]

${yellow('To install a new (toolset) package to "devDependencies":')}
${cyan('$')} npm install --save-dev [package_name@version]

${yellow('To update a package:')}
${cyan('$')} npm update [package_name@version]

${yellow('To uninstall a package:')}
${cyan('$')} npm uninstall [package_name]

${yellow('For detailed instructions, see')} ${cyan(setupInstructionsUrl)}`
// ...
function ensureYarn() {
  if (process.env.npm_execpath.includes('npm')) {
    console.log(NPM_INFO_MSG);
  }
}
```

This might also make it more obvious to read/edit, since newlines and the like can be embedded directly in the same way it will be displayed